### ISSUE TITLE
Strange dice loss for my own dataset

### ISSUE THREAD

**Issue by mamamadoge** [voxelmorph/voxelmorph#114]
I use voxelMorph with my own dataset (ADNI), but the result seems stange.
after training more than 500 epoch, the dice loss for test dataset is stange.
for the whole brain(with out multi label),the dice loss is about 0.8
However when I use multi-label, to split wm, ventricle and so on. the dice loss is about 0.55
Is that normal? I was confused.

**Comment by mamamadoge** [comment]
In addition, my testing is subject to subject. Not atlas

**Comment by adalca** [comment]
Hi @mamamadoge ,

Please provide more details for us or members of the forum to be able to help you.

Which model are you using?
What is your data? How did you pre-process ig?
How did you train? what loss function, what data process?
Did the loss converge well?

These questions are necessary for us to understand what's going on, and usually in the process you will end up figuring out what went wrong as well! :)
